content/en/docs/multicluster-management/import-cloud-hosted-k8s/import-aws-eks.md

```diff
@@ -1,135 +1,183 @@
+
+## Prerequisites
+
+- You have a Kubernetes cluster with KubeSphere installed, and prepared this cluster as the Host Cluster. For more information about how to prepare a Host Cluster, refer to [Prepare a Host Cluster](../../../multicluster-management/enable-multicluster/direct-connection/#prepare-a-host-cluster).
+- You have an EKS cluster to be used as the Member Cluster.
+
+## Import an EKS Cluster
+
+### Step 1: Deploy KubeSphere on your EKS Cluster
+
+You need to deploy KubeSphere on your EKS cluster first. For more information about how to deploy KubeSphere on EKS, refer to [Deploy KubeSphere on AWS EKS](../../../installing-on-kubernetes/hosted-kubernetes/install-kubesphere-on-eks/#install-kubesphere-on-eks).
+
+### Step 2: Prepare the EKS Member Cluster
+
+1. In order to manage the Member Cluster from the Host Cluster, you need to make `jwtSecret` the same between them. Therefore, get it first by executing the following command on your Host Cluster.
+
+   ```bash
+   kubectl -n kubesphere-system get cm kubesphere-config -o yaml | grep -v "apiVersion" | grep jwtSecret
+   ```
+
+   The output is similar to the following:
+
+   ```yaml
+   jwtSecret: "QVguGh7qnURywHn2od9IiOX6X8f8wK8g"
+   ```
+
+2. Log in to the KubeSphere console of the EKS cluster as `admin`. Click **Platform** in the upper left corner and then select **Clusters Management**.
+
+3. Go to **CRDs**, input `ClusterConfiguration` in the search bar, and then press **Enter** on your keyboard. Click **ClusterConfiguration** to go to its detail page.
+
+   ![search-config](/images/docs/multicluster-management/import-cloud-hosted-k8s/import-gke/search-config.png)
+
+4. Click the three dots on the right and then select **Edit YAML** to edit `ks-installer`. 
+
+   ![click-edit](/images/docs/multicluster-management/import-cloud-hosted-k8s/import-gke/click-edit.png)
+
+5. In the YAML file of `ks-installer`, change the value of `jwtSecret` to the corresponding value shown above and set the value of `clusterRole` to `member`.
+
+   ```yaml
+   authentication:
+     jwtSecret: QVguGh7qnURywHn2od9IiOX6X8f8wK8g
+   ```
+
+   ```yaml
+   multicluster:
+     clusterRole: member
+   ```
+
+   {{< notice note >}}
+
+   Make sure you use the value of your own `jwtSecret`. You need to wait for a while so that the changes can take effect.
+
+   {{</ notice >}}
+
+### Step 3: Create a new kubeconfig file
+
+1. [Amazon EKS](https://docs.aws.amazon.com/eks/index.html) doesn’t provide a built-in kubeconfig file as a standard kubeadm cluster does. Nevertheless, you can create a kubeconfig file by referring to this [documentation](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html). The generated kubeconfig file will be like the following:
```

Comment:
documentation
->
document

I reckon users can refer to that single page instead of the entire documentation.